plat/kvm/x86/lcpu_helpers.S

```diff
@@ -37,7 +37,7 @@
 #ifndef __PLAT_KVM_X86_LCPU_HELPERS_S__
 #define __PLAT_KVM_X86_LCPU_HELPERS_S__
 
-#include <x86/cpu_defs.h>
+#include <uk/asm/arch.h>
```

Comment:
With the latest change to forbid `arch.h` to be included directly, this would normally break. The only reason it doesn't is because `lcpu_helpers.S` is included by `lcpu_start.S` which *happens* to first include `lcpu.h`, and thus define `__UKARCH_LCPU_H__` and pass the check in `arch.h`. Conclusion: This change must be reverted, please include `cpu_defs.h` as before.